client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -182,6 +185,19 @@ const TabDetailsContent: React.FC<{
         .filter((fullArchetype) => fullArchetype?.review)
         .filter(Boolean);
 
+  const issueReportsQuery = useFetchIssueReports(application.id);
+  const {
+    result: { data, total: totalReportCount },
+    isFetching: isFetchingReports,
+    fetchError: reportsFetchError,
+  } = issueReportsQuery;
+  const currentPageReports = data as (
+    | AnalysisRuleReport
+    | AnalysisIssueReport
+  )[];
+  const minor = currentPageReports.filter((u) => u.effort === 1).length;
+  const critical = currentPageReports.filter((u) => u.effort > 1).length;
```

Comment:
The technique used LGTM.

Just to share -- using https://radash-docs.vercel.app/docs/array/fork this could look like:
```tsx
  const [minor, critical] = fork(currentPageReports, (u) => effort <= 1).map((a) => a.length);
```

Reply:
The changes were made in the commit 'use fork to calculate minor and critical issues.'